packages.json

```diff
@@ -133,7 +133,10 @@
     "41": {
```

Comment:
This can be removed in `40` now, since it will be included in all kinoite versions:

```json
    "40": {
        "include": {
            "all": [],
            "kinoite": []
```

Reply:
But wouldn't that break the support on current Fedora 40 based images?

Reply:
I made a mistake in looking, sorry